### ISSUE TITLE
Could you give some precedure of building tf with bazel?

### ISSUE THREAD

**Issue by 7oud** [fo40225/tensorflow-windows-wheel#117]
I find 1.14 cxx lib in the repo, very helpful! And I want to build tf2 on Windows, Could you give some advice for building tf with bazel on Windows, it seems quite difficult. Or May I build tf2 using cmake?

**Comment by fo40225** [comment]
`bazel build --config=opt //tensorflow:tensorflow_cc`

https://www.tensorflow.org/install/source_windows

**Comment by 7oud** [comment]
yes, I have followed this guide and build tf1.13.1 lib for windows, but lots of problems including no export symbols, only .so file was built and so on, Is tf1.14 fix these problems?

**Comment by fo40225** [comment]
Until Tensorflow 1.14 did support building cpp lib with bazel.

**Comment by 7oud** [comment]
thanks!!! the tf1.14 c++ lib for windows in the repo is built with vs2015 or vs2017?

**Comment by fo40225** [comment]
VS2019 16.1
